.sync.yml

```diff
@@ -5,6 +5,7 @@
 .travis.yml:
   beaker_sets:
     - centos6-64
+    - 'centos7-64{image=centos:7.6.1810}'
```

Comment:
This specific syntax isn't going to work with our modulesync template. https://github.com/theforeman/foreman-installer-modulesync/pull/121 is intended to take care of this.